cpp/ps0006/cpp_ps0006_solution.cpp

```diff
@@ -36,17 +38,31 @@ int main(void)
 
 void Time(int end, int start, int steps)
 {
+    if (start == end && steps == 0)
+    {
+        cout << end << endl;
+        return;
+    } // try to get rid of that case to implement your code comfortably
 
-    if (start <= end && steps > 0 && end < 100)
+    if (start < end && steps > 0)
     {
         for (int i = start; i <= end; i += steps)
         {
             cout << i;
         }
     }
 
+    else if (start > end && steps < 0)
+    {
+        for (int i = start; i >= end; i += steps)
+        {
+            cout << i;
+        }
+    } // you didn't validate that case
+
     else
     {
-        cout << "error can't start from" << end << "to" << start << "with steo" << steps << endl;
+        cout << "error can't start from" << end << "to" << start << "with step" << steps << endl;
     }
-}
\ No newline at end of file
+} // for bonus you tried but not correct
+// good work 👍 7/10

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors while compilation
- [ ] no runtime errors, bugs or warnings while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
- [x] used `for` loop in a right way to to count up and down

you can check the [problem solution](https://onlinegdb.com/vdj-VXYv7) for more